src/App.js

```diff
@@ -16,22 +17,112 @@ What should a cart item look like? 🤔
 
 console.log(initialStoreItems)
 
-export default function App() {
+function App() {
   // Setup state here...
+  const [storeItems, setStoreItems] = useState(initialStoreItems)
+  const [cartItems, setCartItems] = useState([])
+
+  function addToCart(itemToAdd) {
+    // using the find() to find the first item that matches my itemToadd.id
+    if (cartItems.find(item => item.id === itemToAdd.id)) {
```

Comment:
Nice way to condition your add function based on whether the item is already in the cart